### DISCUSSION TITLE
Outdated documentation to download.

### DISCUSSION THREAD

**Discussion by benabhi** [pallets/flask#4849] [Q&A]
Hello everyone!, I don't know if this is the right place to post this, but I wanted to let you know in the download section of the official documentation (https://readthedocs.org/projects/flask/downloads/) it is only available up to 1.1.x but there is no download for version 2.2.x. I really like the print option and I know there are many others like me who like to read on paper.

**Comment by tachyondecay** [comment]
These are not being generated anymore, see: https://github.com/pallets/flask/issues/4604#issuecomment-1133510733

You can download the docs (by cloning this repo) and then run Sphinx to generate a printable version yourself.
